Sources/tuist/TuistApp.swift

```diff
@@ -13,6 +13,10 @@ private enum TuistServer {
             try? ProcessEnv.setVar(Constants.EnvironmentVariables.verbose, value: "true")
         }
 
+        if CommandLine.arguments.contains("--quiet") {
```

Comment:
> We’d throw an error if you passed both.

Would we like to throw an error at the top level here in `TuistServer`?

Reply:
Yes, I think so 👍

Reply:
I was playing with `swift` CLI and it appears that when you pass both `--verbose` and `--quiet` flags, the `--verbose` takes precedence. 

```sh
Demo ❯ swift build --quiet --verbose
Building for debugging...
Write auxiliary file ~/Demo/.build/arm64-apple-macosx/debug/swift-version-7BB6C6736DFA3E78.txt
Write auxiliary file ~/Demo/.build/arm64-apple-macosx/debug/Demo.build/sources
/Applications/Xcode-16.0.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -module-name Demo -emit-dependencies -emit-module -emit-module-path ... <MORE VERBOSE LOGS>
~/Demo/Sources/Demo/Demo.swift:2:9: warning: initialization of variable 'greetings' was never used; consider replacing with assignment to '_' or removing it
1 | func foo() {
2 |     var greetings = "Hello, World!"
  |         `- warning: initialization of variable 'greetings' was never used; consider replacing with assignment to '_' or removing it
3 | }
4 |
Build complete! (0.41s)
```

Just wanted to make sure that we still would like to throw error instead? 🤔

Reply:
I'd throw an error personally but don't feel strongly about it